DQM and DAQ update

mexanick opened the pull request:
* added DQM canvases
* corrected slot tables
* added online DQM module
* added DAQ registers to GLIB address table
* added DAQ functionality to python script

jsturdy commented:
* There is a compile error with the DQM:
   * dqmCanvases.cxx seems to be missing (I couldn't find it in my checkout of your branch, nor could I find it in github on your branch (release-v2)
```
In file included from /home/sturdy/gemdev/gem-daq-code/gem-light-dqm/dqm-root/../../gem-light-dqm/dqm-root/src/common/main.cxx:2:
/home/sturdy/gemdev/gem-daq-code/gem-light-dqm/dqm-root/../../gem-light-dqm/dqm-root/src/common/treeReader.cxx:523:31: error: dqmCanvases.cxx: No such file or directory
In file included from /home/sturdy/gemdev/gem-daq-code/gem-light-dqm/dqm-root/../../gem-light-dqm/dqm-root/src/common/main.cxx:2:
/home/sturdy/gemdev/gem-daq-code/gem-light-dqm/dqm-root/../../gem-light-dqm/dqm-root/src/common/treeReader.cxx: In member function ‘void gemTreeReader::fillHistograms()’:
/home/sturdy/gemdev/gem-daq-code/gem-light-dqm/dqm-root/../../gem-light-dqm/dqm-root/src/common/treeReader.cxx:349: error: ‘printDQMCanvases’ was not declared in this scope
```

I'm going to merge this in (eventually), but will say a few things that I think we should adopt in the future:
 * We should *not* be updating the config XML files when the only things that change are the values, the only changes that need to really come into the version control are when parameters/config options are added/removed.  I guess this will mean adding them to the .gitignore file
 * In the past (and mostly still) I was opposed to linking the gem online software with the gem dqm software, I realize we want to have some "online DQM" and this is a way to do it quickly, but I think we should re-evaluate at some point once the framework is stable/running.
  * In principle, the DQM *only* needs to know which file to read and some basics about system configuration.

mexanick commented:
I added  the missing file, now it compiles. Considering XML configs - yeah, I agree, that operational configs should stay unchanged, i.e. added to gitignore. I think other XML cfgs, like address tables, should remain in the list of non-ignorant files. Do you know how I can remove xml modifications from current commit (except rolling back the files themselves)? As for the Online DQM part - we can discard it, if you want. However, we need to rewrite the data parker to use with new firmware and AMC13 and eventually discard old data parker at all, so I decided to include for the moment everything we have. Let me know your opinion.

jsturdy commented:
OK, I'll merge this in (and I'm working on the .gitignore situation right now :)
Definitely agree about the different xml files...
Let's leave the DQM as is for now, but keep it in mind for the future
